packages/static-wado-util/lib/assertions/assertAeDefinition.js

```diff
@@ -0,0 +1,31 @@
+const assert = require("assert").strict;
+const aeConfig = require("../aeConfig");
+
+/**
+ * Checks whether there is an ae definition into aeConfig or not.
+ * In case result is false and there is a valid array of errorMessages, then it will throw an exception.
+ *
+ * @throws Exception if there is valid array of errorMessages is passed and result is false
+ * @param {*} params object to check if there is a property name as aeStr or not.
+ * @param {*} aeStr string reference
+ * @param {*} errorMessages array of error messages for missing ae and missing ae def. Where first index is error string for missing ae and second index is error message for missing ae def.
+ * @returns boolean
+ */
+module.exports = function assertAeDefinition(params, aeStr, errorMessages = []) {
```

Comment:
Can you default the error messages?  Would be nice not to create those/have a long setup in this?

Reply:
decided  (async) to go with this solution